From 23ad7a72d33609f0ee41f436834f0e2a19696bc2 Mon Sep 17 00:00:00 2001
From: Philip Balister <philip@balister.org>
Date: Fri, 5 Aug 2016 13:27:55 -0400
Subject: [PATCH 1/2] Fix lib64 detection for cross compiling.

We can't use information from the build machine to determine the target
configuration.

Signed-off-by: Philip Balister <philip@balister.org>
---
 cmake/Modules/GrPlatform.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/Modules/GrPlatform.cmake b/cmake/Modules/GrPlatform.cmake
index a2e4f3b..0e9891a 100644
--- a/cmake/Modules/GrPlatform.cmake
+++ b/cmake/Modules/GrPlatform.cmake
@@ -29,11 +29,11 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
     set(LINUX TRUE)
 endif()
 
-if(LINUX AND EXISTS "/etc/debian_version")
+if(NOT CMAKE_CROSSCOMPILING AND LINUX AND EXISTS "/etc/debian_version")
     set(DEBIAN TRUE)
 endif()
 
-if(LINUX AND EXISTS "/etc/redhat-release")
+if(NOT CMAKE_CROSSCOMPILING AND LINUX AND EXISTS "/etc/redhat-release")
     set(REDHAT TRUE)
 endif()
 
-- 
2.5.5

